src/wast-parser.cc

```diff
@@ -1505,16 +1505,10 @@ Result WastParser::ParseImportModuleField(Module* module) {
       Consume();
       ParseBindVarOpt(&name);
       auto import = MakeUnique<FuncImport>(name);
-      if (PeekMatchLpar(TokenType::Type)) {
-        import->func.decl.has_func_type = true;
```

Comment:
Is the new code still setting this flag somewhere? is it needed?

Reply:
`ParseTypeUseOpt` takes care of setting that:

https://github.com/WebAssembly/wabt/blob/48405ebcbdd85f7d48895560855fcf654d87eeb0/src/wast-parser.cc#L1751-L1755